### PR TITLE
zh-trans: /docs/tutorials/k8s101.md

### PR DESCRIPTION
**Pull request by pigletfly**
chinese translation for /docs/tutorials/k8s101.md

### CODE REVIEW

File: content/zh/docs/tutorials/k8s101.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+<!--
+## Pods
+
+In Kubernetes, a group of one or more containers is called a _Pod_. Containers in a Pod are deployed together, and are started, stopped, and replicated as a group.
+
+For more information, see [Pods](/docs/concepts/workloads/pods/pod/).
+-->
+## Pods
+
+在 Kubernetes 中，一组一个或者多个容器被称为  _Pod_ 。在一个 Pod 里的容器是在一起部署的，它们作为一组一起启动、停止和伸缩容。
```
Comment (markthink) [inline content/zh/docs/tutorials/k8s101.md]: 它们作为一组一起启动、停止和伸缩。

File: content/zh/docs/tutorials/k8s101.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+{{< codenew file="pods/simple-pod.yaml" >}}
+
+A Pod definition is a declaration of a _desired state_.  Desired state is a very important concept in the Kubernetes model.  Many things present a desired state to the system, and Kubernetes' ensures that the current state matches the desired state.  For example, when you create a Pod and declare that the containers in it to be running. If the containers happen not to be running because of a program failure, Kubernetes continues to (re-)create the Pod in order to drive the pod to the desired state. This process continues until you delete the Pod.
+
+For more information, see [Kubernetes Design Documents and Proposals](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/README.md).
+-->
+#### Pod 定义
+
+最简单的 Pod 定义描述了 Deployment 中的单个容器。例如，一个 nginx web 服务器的 Pod 可能被定义为：
```
Comment (markthink) [inline content/zh/docs/tutorials/k8s101.md]: 一个 nginx web 服务器 Pod

File: content/zh/docs/tutorials/k8s101.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+
+{{< codenew file="pods/simple-pod.yaml" >}}
+
+一个 Pod 定义是对预期状态的声明。预期状态在 Kubernetes 模型中是一个非常重要的概念。在这个系统中的很多东西都呈现为预期的状态，Kubernetes 保证现在的状态和预期的状态保持一致。例如，当您创建了一个 Pod 并且声明了将要在里面运行的容器。如果这些容器因为程序故障没有运行，Kubernetes 将继续创建或者重新创建这个 Pod 以使这个 Pod 达到期望状态。这个过程会一直持续直到您删除这个 Pod。
```
Comment (markthink) [inline content/zh/docs/tutorials/k8s101.md]: 一个 Pod 定义是对预期状态的声明
A Pod definition is a declaration of a _desired state_.
建议译为：
Pod 是对预期状态的声明的定义

File: content/zh/docs/tutorials/k8s101.md [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+kubectl get pods
+```
+
+<!--
+On most providers, the Pod IPs are not externally accessible. The easiest way to test that the pod is working is to create a busybox Pod and exec commands on it remotely. For more information, see [Get a Shell to a Running Container](/docs/tasks/debug-application-cluster/get-shell-running-container/).
+
+If the IP of the Pod is accessible, you can access its http endpoint with wget on port 80:
+-->
+
+在大多数提供商上，Pod IP 无法从外部访问。测试 Pod 是否工作的最简单方法是创建 busybox Pod 并且远程执行 exec 命令。有关更多信息，请参阅 [获取正在运行的容器的Shell](/docs/tasks/debug-application-cluster/get-shell-running-container/).
```
Comment (markthink) [inline content/zh/docs/tutorials/k8s101.md]: On most providers,在大多数提供商上
建议译为：
对于大多数服务提供商

Comment (markthink) [inline content/zh/docs/tutorials/k8s101.md]: 创建 busybox Pod => 创建一个 busybox Pod

File: content/zh/docs/tutorials/k8s101.md [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+
+In this example you can create a Redis Pod with a named volume, and a volume mount that defines the path to mount the Volume.
+-->
+#### 卷
+
+这对于简单的静态 Web 服务器来说非常棒，但是持久存储呢？
+
+容器文件系统只在容器中存在。因此，如果您的应用程序的状态需要在重定位，重新启动和崩溃中存活，您将需要配置一些持久存储。
+
+在此示例中，您可以创建具有命名卷的 Redis Pod，以及定义安装卷的路径的卷装入。
```
Comment (markthink) [inline content/zh/docs/tutorials/k8s101.md]: In this example you can create a Redis Pod with a named volume, and a volume mount that defines the path to mount the Volume.
建议译为：
在此示例中，您将创建一个具有数据卷的 Redis Pod,并将数据卷挂载到指定的路径。

**Comment by zhangxiaoyu-zidif** [comment]
@tengqm , never mind, if this PR will be merged into release-1.12, that's fine.

**Comment by tengqm** [comment]
@zhangxiaoyu-zidif Don't think so. The 'master' branch for website repo is for 1.12. It will be archived soon when 1.13 is out (then master becomes 1.13). The Chinese version proposed to release-1.12 should match that English version IIUC.

**Comment by zhangxiaoyu-zidif** [comment]
@tengqm we change our translation policy.
now we base on release-1.12 to translate, and then we will diff the 1.12 and 1.13 , we only translate the diff and merge it to release-1.13. we do not care about the master branch. Here the author should change his dest branch to release-1.12 not master.

**Comment by tengqm** [comment]
@zhangxiaoyu-zidif Exactly. If this PR targets release-1.12 in the Chinese translation repo, and in the release-1.12 English repo, these articles disappeared, we will never be able to sync this two versions. The removal of the two files mentioned was merged into En master, which is the official 1.12 version under maintenance. So my suggestion would be that we save the effort to commit this to Cn 1.12. It will cause out of sync anyway.

**Pull request merged**